src/main/java/picard/sam/FilterReadsByFlowCellLocation.java

```diff
@@ -0,0 +1,106 @@
+package picard.sam;
+
+import htsjdk.samtools.*;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import picard.cmdline.CommandLineProgram;
+import picard.cmdline.StandardOptionDefinitions;
+import picard.cmdline.programgroups.ReadDataManipulationProgramGroup;
+import htsjdk.samtools.util.Log;  // Added this import
+import java.io.File;
+import java.io.IOException;
+
+@CommandLineProgramProperties(
+        summary = "Filters out reads with specific flowcell coordinates",
+        oneLineSummary = "Removes reads from specific flowcell positions from BAM/CRAM files",
```

Comment:
Filter out reads from 'a' specific flowcell position.   It only does 1.

I would probably add a toplevel comment describing what problem this works around also because it's pretty specific.